templates/web/pages/rsvp.tmpl

```diff
@@ -122,11 +119,77 @@
               </div>
             </div>
 
-            <p>Please join our Slack workplace to meet other hackers and hear announcements from the team!
-            At this time, we are not able to provide buses or travel reimbursements. We encourage joining the #carpool channel on Slack to find others from your school.</p>
+            <h4>Address (for swag)</h4>
```

Comment:
If I was a hacker this wouldn't be super clear, maybe we can be like "Your Shipping Address (for swag!)"

Reply:
Yeah I agree Ima make this change.